application/app/src/main/java/com/example/liftvectr/ExerciseHistory.java

```diff
@@ -0,0 +1,106 @@
+        // Perform ItemSelectedListener
+        bottomNavigationView.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
+            @Override
+            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
+                switch(item.getItemId()) {
+                    case R.id.create:
+                        startActivity(new Intent(getApplicationContext()
+                                , MainActivity.class));
+                        overridePendingTransition(0, 0);
+                        return true;
+                    case R.id.view:
+                        return true;
+                }
+                return false;
+            }
+        });
+
+        /*
+        ArrayAdapter<CharSequence> adapter = ArrayAdapter.createFromResource(this, R.array.exercises_array,
+                android.R.layout.simple_spinner_item);
+        adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+        exerciseListSpinner.setAdapter(adapter);
+
+        exerciseListSpinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+            @Override
+            public void onItemSelected(AdapterView<?> adapterView, View view, int pos, long l) {
+                String selectedExercise = (String) exerciseListSpinner.getSelectedItem();
+                Log.i("ExerciseHistory: ", selectedExercise);
+
+                switch (selectedExercise) {
+                    case "Bench Press":
+                        break;
+                    case "Squat":
+                        break;
+                    case "Deadlift":
+                        break;
+                    default:
+                        break;
+                }
+            }
+
+            @Override
+            public void onNothingSelected(AdapterView<?> adapterView) {
+                Log.i("ExerciseHistory: ", "Nothing is selected.");
+            }
+        });
+         */
+
+        //Populating Exercises
+        exerciseViewModel = new ViewModelProvider(this).get(ExerciseViewModel.class);
+        exerciseViewModel.getAllExercises().observe(this, exercises -> {
+            System.out.println("An exercise has been added or deleted! Refresh the ui with the list of exercises here!");
+
+            LiveData<List<Exercise>> savedExercises = exerciseViewModel.getAllExercises();
+            List<String> exerciseData = new ArrayList<String>();
+            for (int i =0; i < savedExercises.getValue().size(); i++) {
+                exerciseData.add( (String) savedExercises.getValue().get(i).getType()
+                        + " " + (String) savedExercises.getValue().get(i).getDate().toString() );
+            }
+
+            ArrayAdapter<String> adapter2 = new ArrayAdapter<String>(this, android.R.layout.simple_list_item_1
+            , exerciseData);
+            exerciseList.setAdapter(adapter2);
+
+        });
```

Comment:
Create transition functions to send data to ChartDisplay when an exercise in the list is pressed, and return to the ExerciseList. See the code examples for transitions between MainActivity ->ChartDisplay